packages/grafana-ui/src/components/DataLinks/DataLinkInput.tsx

```diff
@@ -172,12 +179,15 @@ export const DataLinkInput: React.FC<DataLinkInputProps> = memo(
                   {({ ref, style, placement }) => {
                     return (
                       <div ref={ref} style={style} data-placement={placement}>
-                        <DataLinkSuggestions
-                          suggestions={stateRef.current.suggestions}
-                          onSuggestionSelect={onVariableSelect}
-                          onClose={() => setShowingSuggestions(false)}
-                          activeIndex={suggestionsIndex}
-                        />
+                        <CustomScrollbar scrollTop={getScrollTop()} autoHeightMax="190px">
```

Comment:
@oscarkilhed can you make the popover bigger? 190px feels and looks super tiny when interacting with the suggestions; 300px looks better IMO

Reply:
Good point, 190px was very good in terms of not going outside of the viewport even with a very small viewport. But it does make the "normal" use case a bit worse. The popper doesn't compress, so with a very small viewport there will be some suggestions hidden. Unless we shrink the CustomScrollbar when the viewport is tiny.